src/pages/_app.tsx

```diff
@@ -5,6 +5,7 @@ import { trpc } from "client/lib/trpc";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { httpBatchLink } from "@trpc/client";
 import { useState } from "react";
+import "../styles/globals.css";
```

Comment:
minor: we should ideally decide on an import convention - using absolute or relative paths. For e.g. there are two ways to import this:
import "../styles/globals.css"; OR
import "@/styles/globals.css";   // src is mapped to "@" in tsconfig

@codetheweb : do you have an opinion/preference?

Reply:
I'll update the file as soon as we confirm this!

Reply:
"Absolute" paths are preferred for anything > 1 folder away, but the path aliases are a little wonky--will create a new issue to fix them.

Reply:
https://github.com/sfbrigade/compass/issues/56